packages/js/manifests/wrap/src/compare.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  MethodDefinition, PropertyDefinition,
+} from ".";
+
+export function compareSignatures(
```

Comment:
I'd refactor this to compare signatures of two `MetodDefinition`s instead of two `MethodDefinition[]`s. Just a sylistic preference for easier future reusability.